target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -54,6 +53,14 @@ case "$FIRMWARE" in
 	spectrum,sax1v1k)
 		caldata_extract_mmc "0:ART" 0x1000 0x20000
 		;;
+	zyxel,nbg7815)
+		caldata_extract "0:art" 0x1000 0x20000
+		label_mac=$(get_mac_label)
+		ath11k_patch_mac $(macaddr_add $label_mac 2) 0
+		ath11k_patch_mac $(macaddr_add $label_mac 3) 1
```

Comment:
I think that the order should be:
- phy0 - mac+3 (5GHz)
- phy1 - mac+2 (2.4GHz)
- phy2 - mac+4 (5GHz 2nd)

Is it possible to run 2 separate 5GHz radios on the OEM firmware?

Reply:
Hi,

ty for looking into this. I never thought about if there is a "rule" for assigning MAC's. I would understand it if there is a Wifi MAC printed on the device. But I'll change the numbering/order as you suggested to comply with the rules.

Zyxel is using a script just incrementing to MAC from bootloader env. I knew. So I've just flashed vendor firmware to check what is comming up (for my device):

A0 = LAN
A1 = WAN
A2 = 2G
A3 = 5G

On stock fw we have only one 5G radio exposed to clients. So numbering has to be 3,2,4.